SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerMultiPossessionTest.cpp

```diff
@@ -67,18 +67,20 @@ void ACrossServerMultiPossessionTest::PrepareTest()
 			return ATestPossessionPlayerController::OnPossessCalled == GetNumRequiredClients();
 		},
 		nullptr,
-		[this](float DeltaTime) {
+		[this](float) {
```

Comment:
As per previous PR. Why do we need the DeltaTime lambda here? (Instead of the other parameter which doesnt take a float)